ballerina-tests/tests/configuration_request_limit_config_test.bal

```diff
@@ -109,7 +109,7 @@ service /requestPayloadLimit on lowPayloadLimitEP {
 function testValidUrlLength() {
     http:Client limitClient = checkpanic new("http://localhost:" + requestLimitsTestPort1.toString());
     http:Response|error response = limitClient->get("/requestUriLimit/validUrl");
```

Comment:
We could do as follows:
```ballerina
http:Response response = check limitClient->get("/requestUriLimit/validUrl");
// do other validations
```

Reply:
Since this is not an anti-pattern, kept the test cases as it is to handle errors